extensions/smallrye-graphql-client/deployment/src/test/java/io/quarkus/smallrye/graphql/client/deployment/model/TestingGraphQLApi.java

```diff
@@ -25,15 +24,17 @@ public List<Person> people() {
         return List.of(person1, person2);
     }
 
-    @Inject
-    CurrentVertxRequest request;
-
     /**
      * Returns the value of the HTTP header denoted by 'key'.
      */
     @Query
     public String returnHeader(String key) {
-        return request.getCurrent().request().getHeader(key);
```

Comment:
What's the reason for this change? This is a GraphQLApi so it should preferably use only public APIs, which `CurrentVertxRequest` is, but `ContextHelper` isn't. Does this mean that injecting `CurrentVertxRequest` in this case no longer works??

Reply:
I can remember :) I think in certain cases it does not work. Let me double check that...

Reply:
This is something that should work IMO. I remember advising a user on StackOverflow that this is the only way to obtain the headers. How else would we tell users to do it?

Reply:
Well, at this point all header is available in the context. (via unwrap)
We can add methods on SmallRye Context to expose them. Then basically inject SmallRyeContext and get Headers

Reply:
OK I reverted back to the old test. Everything seems fine.